terraform/virtual-machine.tf

```diff
@@ -60,23 +65,60 @@ resource "azurerm_linux_virtual_machine" "vh_ado_agent" {
 }
 
```

Comment:
Can you add a Data Disk please with name of 

VM NAME_DataDisk
64GB Premium SSD

Reply:
done